frontend/justfile

```diff
@@ -40,6 +40,13 @@ wait-up: up
 # Set up user and test site in Plausible
 init: wait-up
     cd .. && ./setup_plausible.sh
+# Recreate the frontend 
+recreate:
```

Comment:
- `recreate` is used by the other projects for Docker management (which the frontend might also need in the future) so let's rename this to `node-recreate`.
- Since the Node projects in the monorepo are closely intertwined this recreate will apply to all of them and should be moved to the `justfile` in the repo root.